context/app/static/js/components/detailPage/provenance/utils.ts

```diff
@@ -8,6 +8,27 @@ export function createProvDataURL(uuid: string, entityEndpoint: string) {
   return `${entityEndpoint}/entities/${uuid}/provenance`;
 }
 
+function findDifference(obj1: object, obj2: object) {
```

Comment:
Why `object` as the types here rather than something like `Record<string, unknown>`? Is it because the later used `merged[key]` is `unknown` and we can only use a type guard for `object`?

Reply:
I've generified this to match `nonDestructiveMerge`, thanks for pointing out the improvement!